api/nakadi-event-bus-api.yaml

```diff
@@ -1024,6 +1024,26 @@ definitions:
         description: |
           Provides ability to set internal Nakadi parameters.
 
+      write_scopes:
+        type: array
+        items:
+          type: string
+        description: |
+          Provides ability to set oauth2 scopes to be able to publish an Event. The event publisher is cheked for
```

Comment:
What do you think of these suggestions:

"This field is used to authorise clients for publishing to this event type. Nakadi only authorises publishers whose token contains at least one of the scopes in this list."